descriptors/ModuleDescriptor-template.json

```diff
@@ -705,7 +705,7 @@
     },
     {
       "id": "finance.transactions",
-      "version": "1.0 2.1"
+      "version": "1.0 3.0"
```

Comment:
t's not supposed to be an interval. You need to list the supported major versions:
"version": "1.0 2.0 3.0"

Reply:
done